pkg/operator/controllers/suite_test.go

```diff
@@ -72,6 +74,8 @@ var _ = BeforeSuite(func() {
 		ErrorIfCRDPathMissing: true,
 	}
 
+	config := &Config{}
```

Comment:
this Config struct is empty, maybe we need to pass some values so that tests will work?